flexget/plugins/api_tvmaze.py

```diff
@@ -0,0 +1,401 @@
+from __future__ import unicode_literals, division, absolute_import
+
+import logging
+from datetime import datetime
+
+from dateutil import parser
+from pytvmaze import get_show
+from pytvmaze.exceptions import ShowNotFound
+from sqlalchemy import Column, Integer, DateTime, String, Unicode, ForeignKey, Numeric, PickleType, func, Table, and_, \
+    or_
+from sqlalchemy.orm import relation
+
+from flexget import db_schema, plugin
+from flexget.event import event
+from flexget.utils.database import with_session
+from flexget.utils.tools import split_title_year
+
+log = logging.getLogger('api_tvmaze')
+
+DB_VERSION = 0
+Base = db_schema.versioned_base('tvmaze', DB_VERSION)
+UPDATE_INTERVAL = 7  # Used for expiration, number is in days
+
+# TODO Person and character classes are not yet implemented in pytvmaze v.1.3.4
+class TVMazePerson(Base):
+    __tablename__ = 'tvmaze_persons'
+
+    maze_id = Column(Integer, primary_key=True)
+    url = Column(String)
+    medium_image= Column(String)
+    original_image = Column(String)
+    name = Column(Unicode)
+    characters = relation(TVMazeCharacter, secondary=persons_characters_table)
+
+class TVMazeCharacter(Base):
+    __tablename__ = 'tvmaze_characters'
+
+    maze_id = Column(Integer, primary_key=True)
+    url = Column(String)
+    medium_image= Column(String)
+    original_image = Column(String)
+    name = Column(Unicode)
+    persons = relation(TVMazePerson, secondary=persons_characters_table)
```

Comment:
This will happen any day IMO. The guy from pytvmaze is on it. And I know the structure via the original API result so I thought I'd save time by entering it already.